src/installer.ts

```diff
@@ -170,6 +175,36 @@ export async function getNode(
   core.addPath(toolPath);
 }
 
+function isLts(versionSpec: string): boolean {
+  return versionSpec.startsWith('lts')
+}
+
+function findLtsVersionFromManifest(
+  versionSpec: string,
+  stable: boolean,
+  candidates: Array<tc.IToolRelease & { lts?: string }>
+): string {
+  const alias = versionSpec.split('lts/')[1]?.toLowerCase();
+
+  if (!alias) {
+    throw new Error(`Unexpected LTS alias '${alias}' for Node version '${versionSpec}'`);
+  }
+
+  core.debug(`LTS alias '${alias}' for Node version '${versionSpec}'`);
+
+  const release = alias === '*'
```

Comment:
As I understand, we support the following syntax:
- `lts/*` - latest LTS
- `lts/<alias>` - specific LTS label

I would suggest to document it shortly via comment.

Reply:
✔️